mix.exs

```diff
@@ -3,8 +3,8 @@ defmodule ExUnitFixtures.Mixfile do
 
   def project do
     [app: :ex_unit_fixtures,
-     version: "0.3.1",
-     elixir: "~> 1.1",
+     version: "0.4.0",
```

Comment:
I appreciate you're thinking of this, but I'd rather do this when I actually release so I'm going to revert it.